Aplib.Tests/Aplib.Core.Tests.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <TargetFramework>net8.0</TargetFramework>
-        <ImplicitUsings>enable</ImplicitUsings>
+        <ImplicitUsings>disable</ImplicitUsings>
```

Comment:
I take it this is because of `Action`?

Reply:
Yes, as we cannot do the using Action = System..... anymore